### ISSUE TITLE
traceback

### ISSUE THREAD

**Issue by artiction** [xenova/chat-replay-downloader#40]
Is there any reason for the script to stop in the middle of a stream? I've been getting lots of these lately. My internet's pretty okay with 10 MBps download speed..
`python tester.py -message_type all https://www.youtube.com/watch?v=Ei8iGpELhUc -output Ei8iGpELhUc1.txt`

Running the script after the stream ends always works fine of course, it's just that some messages are still being sent out after the stream ends, so I'd like to get them as well. Also, I've been trying to find a way to grab the ones that aren't even showing in the live chat replay although that's not the case here.

```
Traceback (most recent call last):
  File "C:\Users\x\Downloads\Compressed\New folder\tester.py", line 639, in <module>
    chat_messages = chat_downloader.get_chat_replay(
  File "C:\Users\x\Downloads\Compressed\New folder\tester.py", line 563, in get_chat_replay
    return self.get_youtube_messages(match.group(1), start_time, end_time, message_type, chat_type, callback)
  File "C:\Users\x\Downloads\Compressed\New folder\tester.py", line 474, in get_youtube_messages
    callback(data)
  File "C:\Users\x\Downloads\Compressed\New folder\tester.py", line 625, in write_to_file
    print_item(item)
  File "C:\Users\x\Downloads\Compressed\New folder\tester.py", line 616, in print_item
    chat_downloader.print_item(item)
  File "C:\Users\x\Downloads\Compressed\New folder\tester.py", line 208, in print_item
    message = emoji.demojize(self.message_to_string(item))
  File "C:\Users\x\Downloads\Compressed\New folder\tester.py", line 199, in message_to_string
    item['author'],
KeyError: 'author'
```

**Comment by artiction** [comment]
Now that i think of it, it probably fails if there are too many live chats.

**Comment by xenova** [comment]
Hi there! This was actually fixed before #32 with this commit: https://github.com/xenova/chat-replay-downloader/commit/da1b70c48a7d6ee81d118076a82662dbf5589364.

You should be able to fix it by just updating. 👍 (https://github.com/xenova/chat-replay-downloader/releases/latest)

**Comment by artiction** [comment]
Alright, then. I'll see to it, thanks!
